frontend/app-development/config/routes.tsx

```diff
@@ -22,62 +23,55 @@ interface IRouteProps {
 }
 
 interface IRoute {
-  path: string;
+  path: RoutePaths;
   exact: boolean;
   activeSubHeaderSelection: TopBarMenu;
-  menu: string;
   subapp: any;
   activeLeftMenuSelection?: string;
   props?: IRouteProps;
 }
 
 export const routes: IRoute[] = [
```

Comment:
What about renaming this to routerRoutes, then we know that this is internal router routes and not routes to somewhere else.
```suggestion
export const routerRoutes: RouterRoute[] = [
```